app/models/post.rb

```diff
@@ -1,16 +1,39 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  validates :title, presence: true
+  validate :title_not_exceed_250_chars
```

Comment:
- [ ] Good job working here but I think there is another way to do it since here in the documentation is different so kindly check it as well as how you can add validation on the [length](https://guides.rubyonrails.org/active_record_validations.html) so please check it 🙏🏻🚀🔥